scripts/build.sh

```diff
@@ -11,13 +11,24 @@ BASE_ARGS="\
     -DENABLE_JIMPLE_FRONTEND=On \
     -DCMAKE_INSTALL_PREFIX:PATH=$PWD/release \
 "
+# Must disable old frontend to enable goto contractor. github issue #1110
+# https://github.com/esbmc/esbmc/issues/1110.
 SOLVER_FLAGS="\
     -DENABLE_BOOLECTOR=On \
     -DENABLE_YICES=Off \
     -DENABLE_CVC4=OFF \
     -DENABLE_BITWUZLA=On \
-    -DENABLE_GOTO_CONTRACTOR=OFF \
+    -DENABLE_GOTO_CONTRACTOR=On \
+    -DACADEMIC_BUILD=Off \
 "
+# Check if academic build and goto contractor options are enabled and warn about soplex.
+if [ "$(echo "$SOLVER_FLAGS" | grep -c '\-DENABLE_GOTO_CONTRACTOR=On')" -gt 0 ] &&
+      [ "$(echo "$SOLVER_FLAGS" | grep -c '\-DACADEMIC_BUILD=On')" -gt 0 ]; then
```

Comment:
I don't think the build.sh script needs to warn if CMake already does.

Reply:
Removed.